caching-service/src/main/resources/caching-log-messages.yml

```
@@ -114,3 +114,11 @@ messages:
         text: "Missing header with certificate."
         reason: "Call executed without valid client certificate."
         action: "Verify ssl context in client application."
+
+    -   key: org.zowe.apiml.cache.notAvailable
+        number: ZWECS703
+        type: WARNING
+        text: "Cache is not available."
+        reason: "Cache is not ready to write add the moment."
```

```suggestion
        reason: "Cache is not ready to write at the moment."
```

Or simply Cache is not ready to write